aws-iam-role-infraci/variables.tf

```diff
@@ -12,10 +12,10 @@ variable "terraform_state_lock_dynamodb_arns" {
   description = "ARNs of the state file DynamoDB tables"
 }
 
-variable "source_account_id" {
+variable "saml_idp_arn" {
```

Comment:
while you are here, i wonder if worth making these like the oidc blocks so you can configure multiple saml idps

Reply:
(could be in a separate PR)